src/config/sentry.js

```diff
@@ -0,0 +1,25 @@
+export const SENTRY_DSN =
+  'https://6558f0f2adb64d6593db72a3905cfe49@o244065.ingest.sentry.io/6353030';
+
```

Comment:
Get from env variables.

Reply:
I moved it to the env variable REACT_APP_SENTRY_DSN. But now, we have to put it in the .env file for deployment. How should I proceed?